pkg/config/validation.go

```diff
@@ -686,6 +692,25 @@ func validateFromHostSyncMappingObjectName(objRef []string, resourceNamePlural s
 	return nil
 }
 
+const (
+	exportKubeConfigBothSecretAndAdditionalSecretsSetError  = "exportKubeConfig.Secret and exportKubeConfig.AdditionalSecrets cannot be set at the same time"
+	exportKubeConfigAdditionalSecretWithoutNameAndNamespace = "additional secret must have name and/or namespace set"
+)
```

Comment:
nit: should we directly define those as error vars?
```suggestion
var (
	errExportKubeConfigBothSecretAndAdditionalSecretsSet = errors.New("exportKubeConfig.Secret and exportKubeConfig.AdditionalSecrets cannot be set at the same time")
	errExportKubeConfigAdditionalSecretWithoutNameAndNamespace = errors.New("additional secret must have name and/or namespace set")
)
```

Reply:
Yeah, we can. It will just require also a small test change, as the test is using these consts.